src/streamlink/plugins/metube.py

```diff
@@ -0,0 +1,58 @@
+import re
+
+from streamlink.plugin import Plugin
+from streamlink.plugin.api import useragents
+from streamlink.stream import HLSStream
+from streamlink import NoStreamsError
+
+
+class MeTube(Plugin):
+
+    _url_re = re.compile(r"""https?://(?:www\.)?metube\.id/
+                         (?P<type>live|videos)/\w+(?:/.*)?""", re.VERBOSE)
+
+    # extracted from webpage source
+    _VOD_STREAM_NAMES = {
+        "3000k": "1080p",
+        "1800k": "720p",
+        "800k": "480p",
+        "300k": "240p"
+    }
+
+    @classmethod
+    def can_handle_url(cls, url):
+        return cls._url_re.match(url) is not None
+
+    def _get_streams(self):
+        stream_type = self._url_re.match(self.url).group("type")
+        m3u8_re = re.compile(r"https?://.+\.m3u8")
```

Comment:
you might want to use a **group name** instead of `group(0)`
so it can be used later

and you might want to check for `"` or `'` so it won't break on the smallest website change

example

https://github.com/streamlink/streamlink/blob/a91e134e873fbe9f1bae41839fb651badb5b0e05/src/streamlink/plugins/sportal.py#L15